### ISSUE TITLE
Validation issue with 232706234 (CL)

### ISSUE THREAD

**Issue by AndresReyesTech** [googlei18n/libphonenumber#1255]
Please read the "guidelines for contributing" (linked above) and fill in the template below.

Country/region affected (e.g., "US"): CL

Example number(s) affected ("+1 555 555-1234"): 232706234

The phone number range(s) to which the issue applies ("+1 555 555-XXXX"): 

The type of the number(s) ("fixed-line", "mobile", "short code", etc.): 

The cost, if applicable ("toll-free", "premium rate", "shared cost"): 

Supporting evidence (for example, national numbering plan, announcement from mobile carrier, news article): **IMPORTANT - anything posted here is made public. Read the guidelines first!** 

[link to demo](http://libphonenumber.appspot.com/phonenumberparser?number=232706234&country=CL)

**Comment by penmetsaa** [comment]
@AnydaSRL,
Thank you for taking the time to improve libphonenumber!

We were not able to connect to +56 232706234. 
Chile has migrated from 8 digit numbering plan to 9 digit one (2 has to be added after area code).
ITU doc: http://www.itu.int/dms_pub/itu-t/oth/02/02/T020200002A0001PDFE.pdf (This is already supported by our library)

23 is not a valid area code. 2 is valid area code of Santiago.
Is there any typo in the number (232706234) you have provided? Is it 22 37 06234 as per migration plan? More details at [PhoneNumberMetadata.xml](https://github.com/googlei18n/libphonenumber/blob/master/resources/PhoneNumberMetadata.xml#L4668)

We are not able to find official evidence for range 2327. 
We need more information to fix this. Please see our [contribution guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md)

**Comment by nabucosound** [comment]
@penmetsaa 
I can **confirm** that the number `232706234` is currently a **valid number**, corresponding to a fixed line in Santiago, Región Metropolitana, Chile. I'll try to elaborate:

* **Before** the 8 to 9 digit migration that happened in Chile, Santiago had an 8 digit format, where all numbers began with the are code prefix "2". Example: 2255 3344 (8 digits)
* **After** the migration, all Santiago were added an extra "2", so in the previous example, the number is now 222 553 344 (9 digits)
* However, due to the increasing demand in numbers, there are now numbers being issued/created with the second digit with "3", as in the number from this issue: 2**3**2 706 234

I am afraid I was unable to find official sources validating this fact, as I am not Chilean and have a limited knowledge on public institutions or publications.

I ended up here because I found the same problem with a phone number from one of my customers, having a "3" as second digit, a number that I can confirm it exists and is working.
